### PR TITLE
Boskos: left-align happy paths

### PR DESCRIPTION
**Pull request by alvaroaleman**
This PR refactors boskos to left-align happy paths in an attempt to make its code more readable. There shouldn't be any change in behaviour. I highly recommend looking at the diff with whitespace changes omitted.

/assign @stevekuznetsov @sebastienvas

### CODE REVIEW

File: boskos/ranch/ranch.go [hunk 1]
```diff
@@ -142,29 +143,35 @@ func (r *Ranch) Acquire(rType, state, dest, owner, requestID string) (*common.Re
...
+			continue
+		}
+		typeCount++
+
+		if state != res.State {
+			continue
+		}
+		if res.Owner != "" {
+			continue
+		}
```
Comment (petr-muller) [inline boskos/ranch/ranch.go]: subjective nit: I feel the `||` would not hurt here to keep the block shorter.

File: boskos/ranch/ranch.go [hunk 2]
```diff
@@ -142,29 +143,35 @@ func (r *Ranch) Acquire(rType, state, dest, owner, requestID string) (*common.Re
...
+		matchingResoucesCount++
+		if matchingResoucesCount >= rank {
```
Comment (petr-muller) [inline boskos/ranch/ranch.go]: Why no `if matchingResourcesCount < rank { continue }` to keep the flow consistent?

File: boskos/ranch/ranch.go [hunk 3]
```diff
@@ -221,26 +225,29 @@ func (r *Ranch) AcquireByState(state, dest, owner string, names []string) ([]com
...
-					return nil, err
-				}
-				resources = append(resources, updatedRes)
-				delete(rNames, res.Name)
-			}
+		if state != res.State {
+			continue
+		}
+		if res.Owner != "" {
+			continue
```
Comment (petr-muller) [inline boskos/ranch/ranch.go]: subjective nit: `||` it

**Comment by alvaroaleman** [comment]
@petr-muller updated

**Comment by ixdy** [comment]
/lgtm

thanks!

**Pull request merged**